### ISSUE TITLE
Unity BreakDown

### ISSUE THREAD

**Issue by zhudaxing** [microsoft/MixedReality-WebRTC#217]
two mouth ago ,I Use 1.0.2 version  ,I test hololens video with unity Editor ,it worked .but today , i play VideoChatDemo in my PC,when peerconnection init,unity will breakDown  ,and have nothing  error log ,every time! I can promise something wrong with peerConnection.InitializeAsync(), I set AutoInitializeOnStart false in PeerConnection Inspercor,and Call   InitializeAsync() by a button,when button click,unity will BreakdDown. but i copy project to anther pc, it is normal.i do not what happend to my PC .I try to update version to 1.0.3 , it not work. Can you help me?

**Comment by zhudaxing** [comment]
two month ago.... sorry

**Comment by zhudaxing** [comment]
i check other Issues ,I found someone have same  problem with me ,because of  Microphone  permission ,unity breakDown  . i hope MR-WebRTC can provider this error for us .beacuse it is hard to find,so hard....

**Comment by djee-ms** [comment]
Yes I know those permission issues are a plague, sorry for that @zhudaxing! We are looking at better audio support (see _e.g._ #99) which should give us more control over audio initializing. Unfortunately at the moment everything is hidden inside the Google WebRTC code so we have little information when things go wrong, and it is difficult to surface an error.
